angular.json

```diff
@@ -712,10 +712,12 @@
         "packager": {
           "builder": "@dynatrace/barista-builders:packager",
           "options": {
-            "releasePackageJson": "package.json",
+            "releasePackageJson": "components/package.json",
             "placeholder": "{{VERSION}}",
-            "styleFolders": ["style"],
-            "assetFolders": ["assets"]
+            "styleFolders": ["components/style"],
+            "assetFolders": ["components/assets"],
+            "lintingRulesTsConfig": "tools/linting/tsconfig.json",
+            "schematicsTsConfig": "components/schematics/tsconfig.json"
```

Comment:
please configure the `outputPath` in the angular.json the output of the tsconfig should be always to `out-tsc` we have this all over the place that the output paths are overridden by the angular.json.

![image](https://user-images.githubusercontent.com/11156362/70973211-7e5df180-20a5-11ea-9753-8ed0123caf23.png)

The output path should be always relative to the workspaceRoot so `dist/components/...`

if you configure this please make an 'additionalCompileTargets` property that is an array.

```json
"additionalCompileTargets": [
  {
    "tsConfig": "tools/linting/tsconfig.json",
    "outputPath": "dist/components/tslint"
  },
  {
    "tsConfig": "components/schematics/tsconfig.json",
    "outputPath": "dist/components/schematics"
  }
]
```